web/css/github/markdown.css

```diff
@@ -676,3 +676,7 @@
 .markdown-body hr {
   border-bottom-color: #eee;
 }
+
+.markdown-body details summary {

```

Comment:
This file is a vendor one (see the comment on top). Modifying it would make maintenance harder (if the file gets updated to a newer version, manual changes will be lost).
Project styles should be added in src/Packagist/WebBundle/Resources/public/css/main.css instead

Reply:
Thank you for answering so quickly. I moved the style to `main.css`.